fastlane/Fastfile

```diff
@@ -276,9 +286,9 @@ REPOSITORY_NAME="WordPress-Android"
     android_finalize_prechecks(options)
     configure_apply(force: is_ci)
     hotfix = android_current_branch_is_hotfix
-    android_bump_version_final_release() unless hotfix
-    download_translations(options) unless hotfix
-    version = android_get_release_version()
+    android_update_metadata(options) unless hotfix
+    android_bump_version_final_release(app: options[:app]) unless hotfix
```

Comment:
The revert of `download_translations(options)` back into `android_update_metadata` is incorrect and need to be fixed. I did NOT fix that one in the `finalize_release` branch in my other PR, so I'll let you fix this one yourself